operatorcourier/cli.py

```diff
@@ -16,7 +17,12 @@ class _CliParser():
     """
 
     def __init__(self):
-        pass
+        # Several different ways of maintaining a single version number for the project
+        # https://packaging.python.org/guides/single-sourcing-package-version/
```

Comment:
Do we really need to include this comment? We've chosen the method to manage the version.